lib/dao.js

```diff
@@ -1135,12 +1135,7 @@ DataAccessObject._coerce = function (where) {
     }
     // Check there is an operator
     var operator = null;
-    if ('object' === typeof val) {
-      if (Object.keys(val).length !== 1) {
-        // Skip if there are not only one properties
-        // as the assumption for operators is not true here
-        continue;
-      }
+    if (val.constructor === Object) {
```

Comment:
This seems a bit more strict than we need... Isn't `typeof val === 'object' && !Array.isArray(val)` what we are looking for?